hw1/01-selectors.md

```diff
@@ -2,6 +2,33 @@
 
 1. plate
 2. bento
-3.
-4.
-5. ...
+3. #fancy
+4. plate apple#
+5. #fancy pickle
+6. .small
+7. orange.small
+8. bento orange.smallpla
+9. plate, bento
+10. *
+11. plate *
+12. plate + apple
+13. bento ~ pickle
+14. plate > apple
+15. orange:first-child
+16. plate apple,pickle:only-child
+17. .small:last-child
+18. plate:nth-child(3)
+19. bento:nth-last-child(2)
+20. apple:first-of-type
+21. plate:nth-of-type(even)
+22. plate:nth-of-type(2n+3)
+23. plate apple:only-of-type
+24. orange,apple:last-of-type
```

Comment:
This looks similar to line 18, where `orange` and `apple` need to be separated into two different selectors (like `orange:last-of-type`).